apps/mull-api/src/app/channel/inputs/channel.input.ts

```diff
@@ -8,10 +10,17 @@ export class ChannelInput implements Partial<Channel> {
 }
 
 @InputType()
-export class CreateChannelInput implements Partial<Channel> {
+export class CreateEventChannelInput implements Partial<EventChannel> {
   @Field()
   name: string;
 
   @Field()
   rights: number;
 }
+
+@InputType()
+export class CreateDmChannelInput implements Partial<DirectMessageChannel> {
```

Comment:
I know this may seem like I'm bias but I would suggest either have DM everywhere so `CreateDmChannelInput` and `DmChannel` and write somewhere that DM stands for direct messages or have have the full name everywhere: `CreateDirectMessageChannelInput ` and `DirectMessageChannel`